packages/contract/src/contract/helpers.ts

```
@@ -74,20 +74,19 @@ export function encodeStringArgs(abi: Abi, methodObj: AbiMessage, args: any[]):
     return encodedArgs
 }
 
-/** Handle errors returned from contract queries by throwing them
+/** Get errors returned from contract queries
  */
-export function handleContractCallOutcomeErrors(response: ContractCallOutcome, contractMethodName: string): void {
+export function getContractError(response: ContractCallOutcome): string | undefined {
     if (response.output) {
         const out: any = response.output
         if (out.isOk) {
             const responseOk = out.asOk
             if (responseOk.isErr) {
-                throw new ProsopoContractError('CONTRACT.QUERY_ERROR', {
-                    context: { error: responseOk.toPrimitive().err.toString(), contractMethodName },
-                })
+                return responseOk.toPrimitive().err.toString()
             }
         }
     }
+    return undefined
```

Potentially confusing when debugging? Could we throw here or return something other than undefined that explains that we've tried to get the contract error but there isn't one?